### ISSUE TITLE
Android example doesn't run

### ISSUE THREAD

**Issue by kaiguo** [react-native-community/react-native-picker#55]
# Bug

## Environment info

React native info output:

```bash
  React Native Environment Info:
    System:
      OS: Windows 10
      CPU: (8) x64 Intel(R) Core(TM) i7-7700K CPU @ 4.20GHz
      Memory: 13.63 GB / 31.79 GB
    Binaries:
      Yarn: 1.22.0 - C:\Program Files (x86)\Yarn\bin\yarn.CMD
      npm: 6.10.2 - C:\Program Files\nodejs\npm.CMD
```

Library version: master branch

## Steps To Reproduce

1. Clone repo
2. yarn install
3. cd example & yarn start
4. open example/android with Android Studio and run test app

Undefined is not an object (evaluating '_js.PickerIOS.Item)
![image](https://user-images.githubusercontent.com/4424330/75842374-f377c380-5d84-11ea-81e4-602e97fe2b7e.png)

Describe what you expected to happen:
Sample app should run without issue.

**Comment by salmanitb** [comment]
@kaiguo can u share u code???

**Comment by jainkuniya** [comment]
Recently @Naturalclar upgraded example app to RN 0.61.5, can you please help documenting steps to run example help? :)

**Comment by kaiguo** [comment]
I've verified the issue has been fixed with @Naturalclar's recent changes #71, closing : )
